e2e/vpc/constants.go

```diff
@@ -11,13 +11,14 @@
 package vpc
 
 const (
-	vpcZone           = "VPC_ZONE"
-	resourceGroupID   = "RESOURCEGROUP"
-	volumeSize        = 10
-	iops              = "0"
-	volumeName        = "e2e-storage-volume"
-	volumeType        = "vpc-block"
-	generation        = "gt"
-	vpcProfile        = "general-purpose"
-	vpcConfigFilePath = "/src/github.com/IBM/ibmcloud-storage-volume-lib/e2e/config/vpc-config.toml"
+	vpcZone                = "VPC_ZONE"
+	resourceGroupID        = "RESOURCEGROUP"
+	volumeSize             = 10
+	iops                   = "0"
+	volumeName             = "e2e-storage-volume"
+	volumeType             = "vpc-block"
+	generation             = "gt"
```

Comment:
I think gt is not supported yet, are we using this value?

Reply:
Currently, vpc backend is not using this flag but we are setting.